portfolio/src/main/webapp/script.js

```diff
@@ -40,26 +37,48 @@ function addHobby() {
 function addProjectDescription(project) {
   
   const details = {
-    nlp: 'Building language models',
-    ta: 'Held weekly office hours and review sessions for introductory Data Structure and Algorithms class',
-    helmet: 'Integrated speech command to a \"smart\" bike helmet built by a team of 9 friends',
+    Nlp: 'Building language models',
+    Ta: 'Held weekly office hours and review sessions for introductory Data Structure and Algorithms class',
+    Helmet: 'Integrated speech command to a \"smart\" bike helmet built by a team of 9 friends',
     unknown: 'Something is wrong, project doesn\'t exist'
   };
   
   const colors = {
-    nlp: '#6ccfe0',
-    ta: '#f7d36f',
-    helmet: '#e0b9f0',
+    Nlp: '#6ccfe0',
+    Ta: '#f7d36f',
+    Helmet: '#e0b9f0',
     unknown: 'black'
   };
   
   if (!(project in details)) {
     project = 'unknown'; 
   };
   
-  const wrapperName = 'project-description-';
+  const wrapperName = 'projectDescription';
   const wrapper = document.getElementById(wrapperName.concat(project));
   
   wrapper.innerText = details[project]; 
   wrapper.style.backgroundColor = colors[project];
+}
+
+/**
+ * This function enables the highlighting of a selected section from the navigation bar. 
+ * This function is called by the clicking of any one of the boxes in the navigation bar.
+ * Input: section to select
+ * Effect: selected section will have their border shown, and all the other sections will not have the border. 
+ */
+function highlightSection(select) {
```

Comment:
I think "select" is a bit ambiguous. Does it take in ID string? Maybe we can rename it to domIdString or something.

Reply:
Done, changed the argument name to selectedIdString